apps/damap-frontend/src/app/components/layout/layout.component.css

```diff
@@ -366,3 +368,28 @@ svg {
   padding-left: 22px;
   font-weight: 400;
 }
+
+.flex-row-container {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.page-info-icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 18px;
+}
+
+.example-button-container button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.info-box-container {
+  font-size: small;
+  font-weight: 300;
```

Comment:
Maybe you can find a font that fits this case, so we are using the typography system across the board. Look at [this](https://m3.material.io/styles/typography/type-scale-tokens) for inspiration. or use `font-size: smaller;`, this sets the font relative to our main font, while small sets it to an absolute level.